utils/config/daos.yml

```diff
@@ -15,8 +15,8 @@
 # default: daos_server
 #name: daos
 
-# Force different port number to connect to access points.
-# default: 10000
+# Default destination port to use if port not specified in host list addresses.
+# default: 10001
```

Comment:
I agree that we should be consistent by picking one or the other, but I'm curious about the evolution toward port 10001 being the default. Why was it 10000 before and now it's 10001?

Reply:
resulted in the fewest changes, would you prefer me to switch back to 10000?

Reply:
I don't have a particular preference, I'm just concerned about making a change without a specific reason in mind. @johannlombardi, was there a reason port 10000 (or 10001) was chosen? Was there a customer requirement driving it or perhaps it was just that port 10000 was a nice round number and less likely to conflict with existing services?